torchdynamo/config.py

```diff
@@ -18,7 +19,7 @@
 # INFO print compiled functions + graphs
 # WARN print warnings (including graph breaks)
 # ERROR print exceptions (and what user code was being processed when it occurred)
-log_level = logging.WARNING
+log_level = logging.ERROR
```

Comment:
Fun bonus to stem the spam

Reply:
nevermind theres some cursed test

Reply:
IMO warning should be on by default, it just needs to be less chatty